### DISCUSSION TITLE
Script getting values from an other file.

### DISCUSSION THREAD

**Discussion by Barbawesome** [Stellarium/stellarium#3243] [Q&A]
Hello,

I have been playing around with scripting.  I was wondering if there is a way to access data in an other file or does all the information need to be put into the script.  

For example I would like to make a script that could get date, time, position, from a csv or JSON.

Thank you for your time

**Comment by gzotti** [comment]
General file reading is not available due to security concerns. However, you can use include statements to include a data file in JavaScript-compatible syntax. See the *.inc examples in the scripts directory.
